src/libs/migrations/KeyReportActionsDraftByReportActionID.ts

```diff
@@ -27,7 +27,7 @@ export default function () {
                     return resolve();
                 }
 
-                const newReportActionsDrafts: Record<ReportActionsDraftsKey, OnyxEntry<ReportActionsDrafts>> = {};
+                const newReportActionsDrafts: Record<ReportActionsDraftsKey, OnyxEntry<Record<string, ReportActionsDrafts>>> = {};
```

Comment:
```suggestion
                const newReportActionsDrafts: Record<ReportActionsDraftsKey, OnyxEntry<ReportActionsDrafts>> = {};
```

Let's revert this, the structure is wrong (based on my comment above)